lib/ruby_lsp/tapioca/addon.rb

```diff
@@ -60,6 +60,22 @@ def version
 
       sig { params(changes: T::Array[{ uri: String, type: Integer }]).void }
       def workspace_did_change_watched_files(changes)
+        constants = changes.filter_map do |change|
+          path = change[:uri].gsub("file://", "")
```

Comment:
For URIs, never manipulate the strings directly as it may produce incorrect results in certain operating systems.


```suggestion
          path = URI(change[:uri]).to_standardized_path
          next unless path
```